Pipfile

```diff
@@ -1,29 +1,23 @@
 [[source]]
-
 url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
-
 [packages]
-
 django = ">=2.2.0"
 "argon2-cffi" = "*"
 django-compressor = "*"
 "psycopg2" = "*"
 bcrypt = "*"
-
+mysqlclient = "*"
```

Comment:
@leong96 The changes made here in the Pipfile are actually needed to solve a bug @nlMeminger and I are facing. Could you please make a new pull request with only the `mysqlclient` addition?